msal/src/Microsoft.Identity.Client/Features/UsernamePassword/PublicClientApplication.cs

```diff
@@ -33,11 +33,15 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Identity.Core.WsTrust;
+using System;
 
 namespace Microsoft.Identity.Client
 {
+#if !ANDROID_BUILDTIME && !iOS_BUILDTIME && !WINDOWS_APP_BUILDTIME
```

Comment:
This looks good for now but I'd really like to look towards a path for feature flags and having the particular platforms for a feature defined in the props file